index.html

```diff
@@ -546,9 +564,28 @@
 
     <section>
       <h1>Introduction</h1>
-      <p>Security of a WoT System can refer to the security of the Thing Description (TD) itself or 
-      the security of the Thing the Thing Description describes.
-    <section>
+      <p>
+        Security and privacy are crucial requirements for Web of Things:
+        Using WoT-based technologies must not put the integrity of a system or the confidentiality of personal information at risk.
+        This document wants to help implementors and stakeholders in fulfilling these important requirements and provides guidelines for secure and privacy-respecting deployments.
+      </p>
+      <p>
+        Definition-wise, this document builds upon the [[WoT-Architecture]] and the ISO definitions for security [[ISO-IEC-27000]] and privacy ([[ISO-IEC-2382]], [[ISO-IEC-29100]]) it is referring to.
+        The informative guidelines defined here accompany the other deliverables published by the WoT Working Group and especially the normative [[WoT-Architecture]],  [[WoT-Thing-Description]], and [[WoT-Discovery]] documents (also see section [[[#related-documents]]]).
+      </p>
+      <p>
+        <!-- TODO: Revisit document overview -->
```

Comment:
ok, but we have to be careful this is not too redundant.

Reply:
You are right, we could also remove it again. Do you have any suggestions on what should be added to the introduction instead? Would it make sense to take over some parts of the abstract and in turn make the abstract a bit more concise?